lib/sunrise/sunrise.php

```diff
@@ -7,7 +7,7 @@
 	return;
 }
 
-$mu_plugin_dir = defined( 'WPMU_PLUGIN_DIR' ) ? constant( 'WPMU_PLUGIN_DIR' ) : constant( 'WP_CONTENT_DIR' ) . '/mu-plugins';
+$mu_plugin_dir = realpath( __DIR__ . '/../..' );
```

Comment:
I suppose load-ordering issues prevent us from using `WPVIP_MU_PLUGIN_DIR` here. It could be helpful to add a comment to explain the rationale.